config_models/models.py

```diff
@@ -10,10 +10,18 @@
 
 from rest_framework.utils import model_meta
 
-
+# Config model values are cached. The caching backend configuration used has implications for how the
+# values are changed in an active release. When a new value is set, its cached value is deleted.
+# But, depending on the cache backend configuration, the old value may still exist in some caches.
```

Comment:
The amount of time the old value continues to exist _may_ depend on the value of `ConfigurationModel.cache_timeout`.